custom_components/tahoma/sensor.py

```diff
@@ -328,6 +328,12 @@
         key=OverkizState.IO_ELECTRIC_BOOSTER_OPERATING_TIME,
         name="Electric Booster Operating Time",
     ),
+    OverkizSensorDescription(
+        key=OverkizState.CORE_TARGET_CLOSURE,
+        name="Target Closure",
+        native_unit_of_measurement=PERCENTAGE,
+        entity_registry_enabled_default=False,
```

Comment:
@iMicknl It’s disabled by default to not confuse end users.

Reply:
Sorry I wanted to do that indeed, but I missed that line apparently...

Reply:
No problem